src/Stripe.net/Services/Subscriptions/SubscriptionItemPriceDataOptions.cs

```diff
@@ -0,0 +1,52 @@
+// File generated from our OpenAPI spec
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class SubscriptionItemPriceDataOptions : INestedOptions
+    {
+        /// <summary>
+        /// Three-letter <a href="https://www.iso.org/iso-4217-currency-codes.html">ISO currency
+        /// code</a>, in lowercase. Must be a <a href="https://stripe.com/docs/currencies">supported
+        /// currency</a>.
+        /// </summary>
+        [JsonProperty("currency")]
+        public string Currency { get; set; }
+
+        /// <summary>
+        /// The ID of the product that this price will belong to.
+        /// </summary>
+        [JsonProperty("product")]
+        public string Product { get; set; }
+
+        /// <summary>
+        /// The recurring components of a price such as <c>interval</c> and <c>interval_count</c>.
+        /// </summary>
+        [JsonProperty("recurring")]
+        public SubscriptionItemPriceDataRecurringOptions Recurring { get; set; }
+
+        /// <summary>
+        /// Specifies whether the price is considered inclusive of taxes or exclusive of taxes. One
+        /// of <c>inclusive</c>, <c>exclusive</c>, or <c>unspecified</c>. Once specified as either
+        /// <c>inclusive</c> or <c>exclusive</c>, it cannot be changed.
+        /// One of: <c>exclusive</c>, <c>inclusive</c>, or <c>unspecified</c>.
+        /// </summary>
+        [JsonProperty("tax_behavior")]
+        public string TaxBehavior { get; set; }
```

Comment:
Same check here.

Reply:
Same as in `InvoiceItemPriceDataOptions.cs`